app/views/posts/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="w-full m-0 p-0">
+  <% if @posts.empty? %>
+    <%= "No posts" %>
+  <% else %>
+    <%= render partial: "partials/user_posts", locals: {posts: @posts} %>
+  <% end %>
+</div>
+<div class="flex justify-center m-4">
+<%= link_to 'Back to user profile', user_path(@user)%>
+</div>
```

Comment:
**OPTIONAL** - Please note that according to the project requirements, you are not required to have any logic in your view files, this would be done in a later milestone. I suggest having only a placeholder in this file 👍 👍